src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/ReviewViewModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
+{
+    public class ReviewViewModel
+    {
+        public string AccountHashedId { get; set; }
+
+        public string BackLinkUrl { get; set; }
+
+        public IEnumerable<ReviewCohortSummaryViewModel> CohortSummary { get; set; }
```

Comment:
This seems a strange name for this property. Maybe "Cohorts" would be fine?

Reply:
Done.